src/lib/explorer/extensions/commands/addnode.cpp

```diff
@@ -34,7 +36,29 @@ namespace commands {
 console_result addnode::invoke (Json::Value& jv_output,
          libbitcoin::server::server_node& node)
 {
-    jv_output["message"] = IN_DEVELOPING;
+    //jv_output["message"] = IN_DEVELOPING;
+
+    administrator_required_checker(node, auth_.name, auth_.auth);
+    auto& root = jv_output;
+
+    auto address = libbitcoin::config::authority(argument_.address).to_network_address();
+
+    code errcode;
+    auto handler = [&errcode](const code& ec){
+        errcode = ec;
+    };
+
+    if (argument_.command == "add") {
+        node.store(address, handler);
```

Comment:
does store do the connect jobs?

Reply:
yes, this is the same interface function as which is used in the p2p protocol, when the receive ip address list from another p2p node. refer to:  protocol_address::handle_receive_address.